mock/py/mockbuild/podman.py

```diff
@@ -52,35 +35,37 @@ def install_pkgmgmt_packages(self):
         self.exec(cmd)
 
     @traceLog()
-    def export(self, cache_file_name, compress_program):
-        """ export container and compress it  """
-        getLog().info("Exporting container: %s as %s", self.image, cache_file_name)
-        cmd_podman = ["podman", "export", self.container_id]
-        podman = subprocess.Popen(cmd_podman, stdout=subprocess.PIPE)
-        cache_file = open(cache_file_name, "w")
-        cmd_compressor = [compress_program, "--stdout"]
-        compressor = subprocess.Popen(cmd_compressor, stdin=podman.stdout, stdout=cache_file)
-        compressor.communicate()
-        podman.communicate()
-        cache_file.close()
+    def exec(self, command):
+        """ exec command in container """
+        util.do(command, chrootPath=self.buildroot.make_chroot_path(),
```

Comment:
Either the docstring is false or the body of this function. I fail to perceive how this exec command in container.

Reply:
Hmm, the docstring is good now, but... I thought that point of Podman() class is to execute inside container using a podman. But this change the logic of Podman to execute the command using nspawn inside of the directory that was extracted from podman image.

Reply:
Nah, I missed this comment - that is right, and actually the point of this change. As long as we support Mock-in-containers/toolbox, it is not yet an easy thing to unconditionally spawn (nested, either Podman or Nspawn) containers by Mock.  It is though OK to let the automatic image download and mount logic on Podman (works fine in any rootless --privileged container).